projects/ccf-rui/src/app/app.component.spec.ts

```diff
@@ -1,31 +0,0 @@
-import { TestBed, async } from '@angular/core/testing';
```

Comment:
Did you delete the app.component tests???

Reply:
They weren't using shallow-render so I thought they were some default tests that we didn't need. Wasn't looking very closely, sorry. Also they were making my tests fail so I took them out. I can put them back though.

Reply:
Yeah, they should go back and just fix the tests that are failing.

Reply:
Okay no problem